src/main/java/org/kohsuke/github/GHRepositoryBuilder.java

```diff
@@ -169,6 +169,18 @@ public S private_(boolean enabled) throws IOException {
         return with("private", enabled);
     }
 
+    /**
+     * Set template repository as GHRepository.
+     *
+     * @param repositoryTemplate
+     *            GHRepository
+     * @return a builder to continue with building
+     *
+     */
+    public S templateRepository(GHRepository repositoryTemplate) throws IOException {
+        return with("template_repository", repositoryTemplate);
+    }
+
```

Comment:
I would have expected this to be here if we could change the repository template afterwards. Can we?

I looked at the doc and couldn't find a mention of it in the update repository section.